api/build.gradle

```diff
@@ -1,3 +1,6 @@
+ext {
```

Comment:
@tduehr I guess you'll remove that when ALL modules will be running JUnit5 tests.

Reply:
Yup